lib/ast/node.rb

```diff
@@ -177,9 +177,7 @@ def append(element)
     #     p value    # => (integer 1)
     #
     # @return [Array]
-    def to_a
-      children
-    end
+    alias to_a children
```

Comment:
I tried this solution, but it didn't work. Docs for `#children` only exists, docs for `#to_a` is ignored. 
Because alias methods are ignored by yard. https://github.com/lsegal/yard/blob/f16cf4cef9ee1a81c3a3875c2af251ea26d50999/lib/yard/templates/helpers/module_helper.rb#L14